.docker/solana-accounts/mint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Directory containing the accounts
+ACCOUNTS_DIR="./accounts"
+# Amount of tokens to mint for each account (in smallest unit, e.g., lamports if the token has 9 decimals)
+MINT_AMOUNT_USDC=1000000000  # 1000 USDC if the token has 6 decimals
+MINT_AMOUNT_ADEX=1000000000  # 1 ADEX if the token has 9 decimals
+# Path to your keypair file
+AUTHORITY_KEYPAIR=/root/accounts/iguana
+# Path to the fee payer keypair file
+FEE_PAYER_KEYPAIR=/root/accounts/iguana
+# Ensure the solana-test-validator is running and solana-cli is set to use local network
+solana config set --url http://127.0.0.1:8899
+solana config set --keypair /root/accounts/iguana
+# Create new SPL tokens with your keypair as the mint authority
+USDC_TOKEN=$(spl-token create-token --decimals 6 --mint-authority "$AUTHORITY_KEYPAIR" --fee-payer "$FEE_PAYER_KEYPAIR" | grep -o -E 'Creating token [A-Za-z0-9]+' | awk '{print $3}')
+ADEX_TOKEN=$(spl-token create-token --decimals 9 --mint-authority "$AUTHORITY_KEYPAIR" --fee-payer "$FEE_PAYER_KEYPAIR" | grep -o -E 'Creating token [A-Za-z0-9]+' | awk '{print $3}')
+
+echo "Created new USDC token: $USDC_TOKEN"
+echo "Created new ADEX token: $ADEX_TOKEN"
+spl-token create-account "$USDC_TOKEN" --owner "$ACCOUNTS_DIR"/iguana
+spl-token create-account "$ADEX_TOKEN" --owner "$ACCOUNTS_DIR"/iguana
+spl-token mint "$USDC_TOKEN" 100000
+spl-token mint "$ADEX_TOKEN" 100000
+# Iterate over each account file in the directory
+for ACCOUNT_FILE in "$ACCOUNTS_DIR"/*.json; do
```

Comment:
I am fine with having multiple accounts for users created and funded while starting the docker node, but this will cause problems for you in the future while implementing HD wallet since there are multiple addresses/accounts per HD wallet and you will need to update the docker image for those tests. A better pattern is to have one distributor account and this sends coins/tokens to addresses at the start of each test similar to how it's done for ethereum and other coins that use dockerized nodes
https://github.com/KomodoPlatform/komodo-defi-framework/blob/9b0133921057ed8fd8d385df1cc8286b22cb98d8/mm2src/mm2_main/tests/docker_tests/eth_docker_tests.rs#L129
https://github.com/KomodoPlatform/komodo-defi-framework/blob/9b0133921057ed8fd8d385df1cc8286b22cb98d8/mm2src/mm2_main/tests/docker_tests/eth_docker_tests.rs#L149

It's up to you if you want to handle this now or later, but handling it here is the right choice so we don't change the docker image again ref. https://github.com/KomodoPlatform/komodo-defi-framework/pull/2187#discussion_r1706541908

Reply:
@shamardy, all the tests are now working correctly with SPL tokens minted to a single account (I've updated the mint.sh file as you suggested). I believe we can introduce the "filling" methods later, as they are primarily needed to cover the currently implemented test case scenarios.